how-to-guides/data-versioning/notebooks/Compare_model_training_runs_on_dataset_versions.ipynb

```diff
@@ -213,7 +210,7 @@
     "**Note**\n",
     "\n",
     "\n",
-    "Instead of logging data to the public project 'common/quickstarts' as an anonymous user 'neptuner' you can log it to your own project.\n",
+    "Instead of logging data to the public project ```common/data-versioning``` as an anonymous user ```neptuner``` you can log it to your own project.\n",
```

Comment:
```suggestion
    "Instead of logging data to the public project `common/data-versioning` as an anonymous user `neptuner`, you can log it to your own project.\n",
```

Reply:
The Markdown syntax for inline code-formatted text is single backticks